### ISSUE TITLE
getattr- AcceptSuggestion : signature does not have a field item

### ISSUE THREAD

**Issue by hankcs** [telepathylabsai/OpenDF#17]
Dear all,

I'm trying to make the user accept suggestion from a delete event operation. But I got this wired exception. I'm not sure why the graph has to call get item on AcceptSuggestion.

## Dialog 48

**GUID**: `e9b23d87-1b3c-4660-92b0-62d74a445706` **OpenDF**:

 - U1: Mark meeting with Jeff Monday at 4pm
 - A1: Is this good?
 - U2: That's fine. Cancel the Meeting with Dave **before that**
 - A2: I've put that on your calendar. __BREAK Is this what you want me to cancel?

Data:

```
{"dialogue_id": "e9b23d87-1b3c-4660-92b0-62d74a445706", "turns": [{"agent_utterance": {"described_entities": ["entity@126098598"], "original_text": "Is this good?", "tokens": ["Is", "this", "good", "?"]}, "lispress": " CreateEvent( AND( with_attendee( Jeff ) , starts_at( NextDOW( MONDAY ) ) , starts_at( NumberPM( 4 ) ) ) ) ", "program_execution_oracle": {"has_exception": true, "refer_are_correct": true}, "skip": false, "turn_index": 0, "user_utterance": {"original_text": "Mark meeting with Jeff Monday at 4pm", "tokens": ["Mark", "meeting", "with", "Jeff", "Monday", "at", "4", "pm"]}}, {"agent_utterance": {"described_entities": ["entity@1006503236"], "original_text": "I've put that on your calendar. __BREAK Is this what you want me to cancel?", "tokens": ["I", "'ve", "put", "that", "on", "your", "calendar", ".", "Is", "this", "what", "you", "want", "me", "to", "cancel", "?"]}, "lispress": " do( Let( x0 , AcceptSuggestion( ) ) , do( Yield( $ x0 ) , DeleteEvent( AND( starts_at( LT( :start( :item( $ x0 ) ) ) ) , with_attendee( Dave ) ) ) ) ) ", "program_execution_oracle": {"has_exception": true, "refer_are_correct": true}, "skip": false, "turn_index": 1, "user_utterance": {"original_text": "That's fine. Cancel the Meeting with Dave before that", "tokens": ["That", "'s", "fine", ".", "Cancel", "the", "Meeting", "with", "Dave", "before", "that"]}}], "_id": 863}
```

Program:

```
 CreateEvent(
    AND(
       with_attendee(
          Jeff ) ,
       starts_at(
          NextDOW(
             MONDAY ) ) ,
       starts_at(
          NumberPM(
             4 ) ) ) ) 

 do(
    Let(
       x0 ,
       AcceptSuggestion(
          ) ) ,
    do(
       Yield(
          $ x0 ) ,
       DeleteEvent(
          AND(
             starts_at(
                LT(
                   :start(
                      :item(
                         $ x0 ) ) ) ) ,
             with_attendee(
                Dave ) ) ) ) ) 
```

Graph:
![Screen Shot 2022-06-21 at 11 05 05 AM](https://user-images.githubusercontent.com/5326890/174833261-b6afa1de-e165-4c8c-92ee-3ffe6823ceee.png)

**Comment by meron-tl** [comment]
fixed issue in 5548dbf4.

**Comment by hankcs** [comment]
Great, thank you.
